styles/ListItem.module.css

```diff
@@ -0,0 +1,21 @@
+.item_list___without_bottom_border {
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
This is the best place to use the Pseudo-classes to define only the last item does not have a bottom border.